src/centralserver/admin-service/src/main/resources/openapi-definition.yaml

```diff
@@ -1,357 +1,4105 @@
+        - DECIPHER_ONLY
+      example: NON_REPUDIATION
+      format: enum
+      type: string
+    ManagementServices:
+      allOf:
+        - $ref: '#/components/schemas/ServiceProviderId'
+      properties:
+        security_server_id:
+          description: encoded id of the security server that provides the management services
+          example: CS:ORG:111:SS1
+          format: text
+          maxLength: 1279
+          minLength: 1
+          readOnly: true
           type: string
+        security_server_owners_global_group_code:
+          description: the url of the management services
+          example: https://dev.xroad.rocks/managementservice/manage/
           format: text
-          description: central server's node name
-          example: node_0
           minLength: 1
+          readOnly: true
+          type: string
+        service_provider_name:
+          description: name of the service provider
+          example: NIIS
+          format: text
           maxLength: 255
-    SystemStatus:
+          minLength: 1
+          readOnly: true
+          type: string
+        services_address:
+          description: the url of the management services
+          example: https://dev.xroad.rocks/managementservice/manage/
+          format: url
+          minLength: 1
+          readOnly: true
+          type: string
+        wsdl_address:
+          description: the url of the management service wsdl
+          example: https://dev.xroad.rocks/managementservices.wsdl
+          format: url
+          minLength: 1
+          readOnly: true
+          type: string
+      required:
+        - security_server_id
+        - security_server_owners_global_group_code
+        - service_provider_name
+        - services_address
+        - wsdl_address
+        - service_provider_id
+      type: object
+    Member:
+      allOf:
+        - $ref: '#/components/schemas/MemberName'
+        - $ref: '#/components/schemas/MemberClassAndCode'
+      description: x-road member
+      properties:
+        id:
+          description: the encoded string representation of the x-road member id
+          example: FI:GOV:123:ABC
+          format: text
+          maxLength: 1023
+          minLength: 1
+          readOnly: true
+          type: string
+        instance_id:
+          description: Xroad instance id. Cannot contain colons, semicolons, slashes, backslashes, percent signs or control characters
+          example: FI
+          format: text
+          maxLength: 255
+          minLength: 1
+          readOnly: true
+          type: string
+        member_type:
+          $ref: '#/components/schemas/MemberType'
+        subsystem_code:
+          description: Subsystem code. Cannot contain colons, semicolons, slashes, backslashes, percent signs or control characters
+          example: ABC
+          format: text
+          maxLength: 255
+          minLength: 1
+          type: string
+      type: object
+    MemberClass:
```

Comment:
Sort of a generic question: do we need `created_at`, `updated_at`? Those exist in DB but not in UI (at least the old UI). 
Are the included properties selected based on what is visible in the UI? What is our approach: 
a) include only properties our frontend needs 
b) include all existing properties 
c) include a + add those properties we expect would be useful for API clients using some sort of ad-hoc decisions

Reply:
Addressed in https://github.com/nordic-institute/X-Road/commit/c81729792696428efce82833df3337d23e43df7a

Went with option b) for the most objects. We can remove properties that we don't think we need when refining the API for each endpoint.

Some join table timestamps were omitted, and auth certificate, since it would have required more structural changes (currently is directly CertificateDetails, which is a shared object and can't have timestamps)